Docs/source/running_cpp/parameters.rst

```diff
@@ -1468,6 +1468,8 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Higher corner of the output fields (if larger than ``warpx.dom_hi``, then set to ``warpx.dom_hi``). Currently, when the ``diag_hi`` is different from ``warpx.dom_hi``, particle output i
 s disabled.
 
+* ``<diag_name>.write_species`` (`0` or `1`) optional (default `1`)
+   Whether to write species output or not. For checkpoint format, always set this parameter to 1.
 
 * ``<diag_name>.species`` (list of `string`, default all physical species in the simulation)
```

Comment:
if a user puts `<diag_name>.species = ` in their input file - would that serve the same purpose?

Reply:
I tried running with an empty parameter, and get an error
```
ParmParse::addDefn(): no values for definition diag1.species
amrex::Abort::SIGABRT
```

From our (@NeilZaim @lucafedeli88 @EZoni ) discussion earlier this week, we initially thought of having <diag_name>.species = "none". As I started implementing, I found a 0/1 flag to be explicit and easy to implement.

Since, in FullDiagnostics.cpp, we initialize particleDiags using the "species_name" and it would complicate the interface for "none"-species.

Reply:
For strings I think we could in principle pass empty parameters if we put the quotes : `<diag_name>.species = ""`

Although for this specific case this does not work (we get `amrex::Abort::0::ERROR: could not find the ID of product species ''. Wrong name? !!!`)

Reply:
Thank you for checking this. Interesting - well this way it's explicit and default-on is relatively safe.